### DISCUSSION TITLE
How about allowing the user to hide the 'donate' button based on their preference?

### DISCUSSION THREAD

**Discussion by hl2570** [seerge/g-helper#3259] [Ideas]
Hi.

First of all, thank you for developing G-Helper. I really appreciate it.

By default, there’s a 'donate' button located at the bottom left. Even though I made a small donation, the button didn’t disappear automatically. I looked through the settings just in case, but I couldn’t find an option to hide the donate button. I was wondering if it might be possible to add a feature that allows users to hide the button. Or how about making it so the donate button doesn’t appear for users who have already donated?

Thank you for your time, and I hope you stay healthy and have a great day!

**Comment by seerge** [comment]
@hl2570 hello, thanks for a feedback (and a donation). 

App doesn't track you, so it's not aware if you actually donated or not. Button intentionally looks same as any other UI elements in the app, not flashy or prominent. Even if you will remove it - it will just leave an empty spot behind, that doesn't look any better imho.

Thanks.

**Reply by AyanPR** [reply]
Based on what you've said here, the donation button should be subtle, but since I started using G-Helper, a random notification keeps popping up on the donation button.
<img width="284" height="56" alt="image" src="https://github.com/user-attachments/assets/a7442603-c033-45ca-a289-7b7e761a2543" />
